### PR TITLE
Added benchmark npm command

### PR DESCRIPTION
**Pull request by lucamaraschi**
This PR adds a new npm script named `benchmark` which acts as a shortcut to perform benchmarks in fastify based on a provided example file.

The `benchmark` command is wrapping the following actions:
* start the fastify instance for the provided example file
* store the $pid of the node process
* execute `autocannon -c 100 -d 5 -p 10` on `localhost:3000/`
* at the end kills the node process for `$pid`

**Usage**
```sh
npm run benchmark --usingfile={file}
```
where `{file}` is the name of an  example file placed in the `/examples` folder.

**Requirements**
* The example file needs to be placed in the `/examples` folder
* The fastify instance needs to be exposed on port **3000**
* The fastify instance must expose `GET /`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "test": "standard | snazzy && tap test/*.test.js test/*/*.test.js",
     "coverage": "tap --cov --coverage-report=html test/*.test.js test/*/*.test.js",
-    "coveralls": "tap test/*test.js  test/*/*.test.js --cov --coverage-report=text-lcov | coveralls"
+    "coveralls": "tap test/*test.js  test/*/*.test.js --cov --coverage-report=text-lcov | coveralls",
+    "benchmark": "node ./examples/$npm_config_usingfile & pid=$! && ./node_modules/autocannon/autocannon.js -c 100 -d 5 -p 10 localhost:3000/ && kill -9 $pid"
```
Comment (allevo) [inline package.json]: `autocannon` lib exports the "binary" directly. It is not needed to specify the path.
Use `autocannon`directly.

**Comment by mcollina** [comment]
There seems to be an issue on autocannon on node v5. I'm actually 👍  on removing it.

**Comment by allevo** [comment]
This's a great!

Can we thought to a comparison benchmark script too?
I mean a script that runs all example to show the differences

**Comment by delvedor** [comment]
@allevo,
@lucamaraschi and me are working on an awesome script that runs the benchmarks and compares it with the master branch, using [autocannon-compare](https://github.com/mcollina/autocannon-compare) 😄

**Comment by allevo** [comment]
Awesome!
@delvedor let me know if i can help

**Comment by delvedor** [comment]
I've fixed the conflicts and removed node v5 from travis :)

**Pull request merged**